web/themes/custom/fluffiness/templates/node--hospital.html.twig

```diff
@@ -90,6 +91,8 @@
   <h2 {{ attributes.addClass(classes2) }} {{ title_attributes }}>
     <span>{{ label }}</span>
   </h2>
+
+  <h3 style="background-color: {{ testvalue }}"> field colour is {{ testvalue }}</h3>
```

Comment:
using inline styles is not good practice. as you have the parent class already, use this to style everything. or you can assign a class to this h3 as well